volatility3/framework/plugins/linux/malware/malfind.py

```diff
@@ -51,14 +61,36 @@ def _list_injections(
 
         proc_layer = self.context.layers[proc_layer_name]
 
+        dump_size = self.config.get("dump-size", None) or 64
```

Comment:
this and the dump-page one should use the `default` parameter to the Int and Boolean requirements on 40 and 45 instead of this way of setting a default.